src/app/static/src/app/components/adr/ADRIntegration.vue

```diff
@@ -11,13 +11,11 @@
     import adrKey from "./ADRKey.vue";
     import SelectDataset from "./SelectDataset.vue";
 
-    declare const currentUser: string;
-
     export default Vue.extend({
         components: {adrKey, SelectDataset},
         computed: {
-            loggedIn() {
-                return currentUser != "guest"
+            loggedIn(): boolean {
+                return !this.$store.getters.isGuest
```

Comment:
TS seems to have issues with a computed property loggedIn returning a getter from mapGetters so I have left this as is. Let me know if there is a better way of doing this.

Reply:
We have a helper utility, `mapGetterByName` that can help here, although at the moment it's set up to only work with namespaced getters, so will need a small tweak to work with root getters.

This should become:
```
isGuest: mapGetterByName(null, "isGuest"),
loggedIn() {
       return !this.isGuest
}
```

You may also need to explicitly type the component, so:
```
    interface Methods {
        getDatasets: () => void
        fetchADRKey: () => void
    }

    interface Computed {
        isGuest: boolean
        loggedIn: boolean
        key: string | null
    }

    export default Vue.extend<{}, Methods, Computed, {}>({
       ...
```

And to make this work, in `app/utils`, the `mapGetterByName` function needs to become:
```
export const mapGetterByName = <T>(namespace: string | null, name: string): ComputedWithType<T> => {
    return (namespace && mapGetters(namespace, [name])[name]) || mapGetters([name])[name]
}
```